deploy/crds/aws_v1alpha1_awsfederatedaccountaccess_crd.yaml

```diff
@@ -38,20 +38,28 @@ spec:
           type: object
         spec:
           properties:
-            accountReference:
-              description: AccountReference holds the name of the associated Account
-                CR to use
-              type: string
+            awsCredentialSecret:
+              description: AwsCredentialSecret holds the credentials to the cluster
+                account where the role wil be created
+              properties:
+                name:
+                  type: string
+                namespace:
+                  type: string
+              required:
+              - name
+              - namespace
+              type: object
             awsFederatedRoleName:
               description: FederatedRoleName must be the name of a federatedrole cr
                 that currently exists
               type: string
             externalCustomerAWSIAMARN:
-              description: externalCustomerAWSIAMARN holds the external AWS IAM ARN
+              description: ExternalCustomerAWSARN holds the external AWS IAM ARN
```

Comment:
Where is this change coming from?

Reply:
+1 keep it lowercase to match actual field.